python/cugraph/cugraph/community/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources
+    ecg_wrapper.pyx egonet_wrapper.pyx
+    ktruss_subgraph_wrapper.pyx
+    leiden_wrapper.pyx
+    louvain_wrapper.pyx
+    spectral_clustering_wrapper.pyx
+    subgraph_extraction_wrapper.pyx
+    triangle_count_wrapper.pyx
+)
+
+set(linked_libraries cugraph::cugraph)
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX community_
+)
+
+set(targets_using_numpy
```

Comment:
We can remove this variable ( 32-40 ) as it isn't used anymore

Reply:
Right, I forgot to remove it.